stdlib/3/builtins.pyi

```diff
@@ -937,11 +937,25 @@ class SystemExit(BaseException):
     code = 0
 class Exception(BaseException): ...
 class ArithmeticError(Exception): ...
-class EnvironmentError(Exception):
-    errno = 0
-    strerror = ...  # type: str
-    # TODO can this be bytes?
-    filename = ...  # type: str
+if sys.version_info >= (3, 3):
+    class OSError(Exception):
+        errno = 0
+        strerror = ...  # type: str
+        filename = ...  # type: Union[str, bytes, None]
+        if sys.version_info >= (3, 4):
+            filename2 = ...  # type: Union[str, bytes, None]
+        if sys.platform == 'win32':
+            winerror = 0
+    EnvironmentError = IOError = WindowsError = OSError
```

Comment:
D'oh, turns out that mypy doesn't allow class aliases... Should I make them subclasses even though they're not?